### PR TITLE
QDB-2999 - Add support for GetLastError().

### PR DESCRIPTION
**Pull request by mkurdej**
I only return the error message here, but I think that we need to modify the error handling so that the user be able to get the exact error code. It should be an easy change but touching all the exception handling stuff (so lots of almost automatic changes).

### CODE REVIEW

File: Quasardb.Native/qdb_error.cs [hunk 1]
```diff
@@ -53,6 +53,14 @@ public enum qdb_error : uint
...
-        qdb_e_column_not_found = 0xb1000039
+        qdb_e_column_not_found = 0xb1000039,
+        qdb_e_query_too_complex = 0xb2000040,
+        qdb_e_invalid_crypto_key = 0xc2000041,
+        qdb_e_invalid_query = 0xc2000042,
+        qdb_e_invalid_regex = 0xc2000043,
+        qdb_e_unknown_user = 0xc2000044,
+        qdb_e_interrupted = 0xf2000045,
+        qdb_e_network_inbuf_too_small = 0xe2000046,
+        qdb_e_network_error = 0xd2000047
```
Comment (Dwarfobserver) [inline Quasardb.Native/qdb_error.cs]: 👀

Comment (mkurdej) [reply Quasardb.Native/qdb_error.cs]: Yeaaah, it was missing.

File: Quasardb/QdbCluster.cs [hunk 2]
```diff
@@ -77,6 +77,18 @@ public QdbCluster(string uri, string clusterPublicKey, string userName, string u
...
 
+        /// <summary>
+        /// Gets the last API error description.
+        /// <returns>A message describing the error occurred during the last operation.</returns>
+        /// </summary>
+        public string GetLastError()
+        {
+            qdb_error error;
+            qdb_sized_string message;
+            qdb_api.qdb_get_last_error(out error, out message);
```
Comment (Dwarfobserver) [inline Quasardb/QdbCluster.cs]: You can pass NULL instead of error if you don't want it (if .NET allows it).

Comment (mkurdej) [reply Quasardb/QdbCluster.cs]: Right.

Comment (mkurdej) [reply Quasardb/QdbCluster.cs]: But I'm not sure how to do it :/.

Comment (Dwarfobserver) [reply Quasardb/QdbCluster.cs]: Let's keep it like that ^^

**Pull request merged**